config/platforms/platform_solaris.h.in

```diff
@@ -11,7 +11,7 @@
 #define DUK_USE_BYTEORDER 3
 #endif
 #else  /* DUK_F_OLD_SOLARIS */
-#include <ast/endian.h>
+#include <sys/param.h>
```

Comment:
That is correct.  I was trying to be obvious about what was happening, and since the include is encapsulated by standard "has this been referenced yet" checks, it shouldn't hurt.

If you feel strongly, i'm happy to remove that include.
The main necessary condition of the patch is to remove ast/endian.h reference, which as stands kills the build on OmniOS